test/integration/glue_test.py

```diff
@@ -86,6 +86,12 @@ def glue_test_dfs(
             ),
         }
     )
+    # Make everything lowercase
+    glue_test_dfs = {
+        df_name: df.applymap(lambda x: x.lower() if type(x) == str else x)
+        for (df_name, df) in glue_test_dfs.items()
+    }
+
```

Comment:
is this necessary now that you moved where the lower casing is happening in the glue module (which i think is the more correct place to have any of that happen)?

Reply:
The tables added by `glue_test_dfs.update(glue(eia=True, ferc1=True))` will be lowercase because the `glue` function calls `get_plant_map` where I added the lower-casing function. 
However, the tables added before:
```
    glue_test_dfs = {
        "plants_eia_pudl_db": pudl_out.plants_eia860(),
        "util_ids_ferc1_raw_xbrl": get_util_ids_ferc1_raw_xbrl(ferc1_engine_xbrl),
        "util_ids_ferc1_raw_dbf": get_util_ids_ferc1_raw_dbf(ferc1_engine_dbf),
        "plants_ferc1_raw": plants_ferc1_raw(dataset_settings_config),
        "plants_eia_labeled": label_plants_eia(pudl_out),
    }
```
and after:
```
    glue_test_dfs.update(
        {
            "utilities_ferc1_dbf_labeled": label_utilities_ferc1_dbf(
                glue_test_dfs["utilities_ferc1_dbf"],
                glue_test_dfs["util_ids_ferc1_raw_dbf"],
            ),
            "utilities_ferc1_xbrl_labeled": label_utilities_ferc1_xbrl(
                glue_test_dfs["utilities_ferc1_xbrl"],
                glue_test_dfs["util_ids_ferc1_raw_xbrl"],
            ),
        }
    )
```
Are not part of that function and won't get lower-cased. It looks like the later bit says: "more glue test input tables that are compiled from tables already in glue_test_dfs" so I wonder A) are they already lower case because they're derived from tables that are? and B) could I move this lower casing function up to just affect the first few tables added to the `glue_test_dfs` that aren't lower cased already?

Reply:
ah okay! well then yea i think this is a fine fail safe. perhaps we can make a more systematic LOWER EVERYTHING issue after this is done so we can do that more uniformly but i think this is a good solution for now!